Sources/MapboxMaps/Style/StyleManagerProtocol.swift

```diff
@@ -1,507 +1,106 @@
 import Foundation
 
-// swiftlint:disable file_length function_parameter_count
+// swiftlint:disable function_parameter_count
```

Comment:
This violation is actually on line 69 for method

```
func addImage(_ image: UIImage,
                  id: String,
                  sdf: Bool,
                  stretchX: [ImageStretches],
                  stretchY: [ImageStretches],
                  content: ImageContent?) throws
```

Rather than disabling this rule for the rest of the file, let's annotate that specific method with…

```
    // swiftlint:disable:next function_parameter_count
```